entrypoint.sh

```diff
@@ -7,6 +7,7 @@ cd "${GITHUB_WORKSPACE}" \
 
 set -eu
 
+BRANCH_NAME=$(git branch | sed -n -e 's/^\* \(.*\)/\1/p')
```

Comment:
```suggestion
BRANCH_NAME=$(git branch --show-current)
```
would be better.

Reply:
I think it's better to assert BRANCH_NAME is not empty. (it will be empty if executed on detached HEAD.)

Reply:
`git branch --show-current` was added by git 2.22.0 which is not available at melodic era. (It's using melodic docker image to run the script.)
I'll fix it and create a new release.